react-native/WINDOWS.md

```diff
@@ -0,0 +1,47 @@
+# Installation
+
+### Requirements
+
+1. Node
```

Comment:
What about `npm` and `npx`?

Reply:
@Satendra-SR npm comes with the node and npx can be installed via npm, which I have mentioned in the `working with the project` section.